README.md

```diff
@@ -108,6 +108,23 @@ ShopifyApp.configure do |config|
 end
 ```
 
+
+WebhooksManager
+---------------
+
+ShopifyApp can manage your apps webhooks for you (requires ActiveJob). Set which webhooks you require in the initializer:
+
+```ruby
+ShopifyApp.configure do |config|
+  config.webhooks = [
+    {topic: 'carts/update', address: 'example-app.com/webhooks', format: 'json'}
```

Comment:
yeah I made json the default and I am not mentioning it. I used a reverse_merge so someone could spec xml if they really wanted but then they are on their own from there.